.github/workflows/actions/prepare-distribution/action.yml

```diff
@@ -85,6 +85,17 @@ runs:
         Compatible systems include, but are not limited to, \`Ubuntu 20.04+\` or \`Debian 11+\` (Bullseye)).
         EOF
 
+    - name: Fetch the latest version of the unstable tag
+      shell: bash
```

Comment:
This should be only added for unstable releases, something like this should do the trick:
```
if: contains('unstable', inputs.version-name)
```